Policies/Kubernetes/allowed-users/azurepolicy.json

```diff
@@ -0,0 +1,85 @@
+{
+  "properties": {
+    "displayName": "Control which user ID containers are run with in a Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy controls allowed user IDs for containers to run with in a Kubernetes Cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc.",
+    "metadata": {
+      "version": "1.0.0-preview",
+      "category": "Kubernetes PSP",
+      "preview": true
+    },
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        },
+        "allowedValues": [
+          "audit",
+          "deny",
+          "disabled"
+        ],
+        "defaultValue": "audit"
+      },
+      "excludedNamespaces": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Namespace exclusions",
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+        },
+        "defaultValue": ["kube-system", "gatekeeper-system", "azure-arc"]
+      },
+      "runAsUserRule": {
+        "type": "String",
+        "metadata": {
+          "displayName": "User Restriction Rule",
+          "description": "How much restriction on users to apply to the cluster"
```

Comment:
Can you change display name to "Run as user" and description to "RunAsUser controls the user ID that containers are allowed to run with."? 
FYI, took the description from https://kubernetes.io/docs/concepts/policy/pod-security-policy/#users-and-groups